viewer/src/index.ts

```diff
@@ -10,3 +10,7 @@ export { Intersection } from './public/migration/intersection';
 export { Color, Cognite3DViewerOptions, AddModelOptions, SupportedModelTypes } from './public/migration/types';
 export { CadLoadingHints } from './datamodels/cad/CadLoadingHints';
 export { BoundingBoxClipper } from './utilities';
+
+// Export ThreeJS to enable easy import for our users
```

Comment:
But why to do so if it's a peer-dependency?

Reply:
Not sure if it's the right thing to do, but it's what the old viewer did... Seems odd to me as well